libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1592,10 +1744,14 @@ namespace SohImGui {
             ImGui::End();
             ImGui::PopStyleColor();
         }
+
         if (CVar_GetS32("gStatsEnabled", 0)) {
+            if (!statsWindowOpen) {
+                CVar_SetS32("gStatsEnabled", 0);
+            }
             const float framerate = ImGui::GetIO().Framerate;
             ImGui::PushStyleColor(ImGuiCol_Border, ImVec4(0, 0, 0, 0));
-            ImGui::Begin("Debug Stats", nullptr, ImGuiWindowFlags_NoFocusOnAppearing);
+            ImGui::Begin("Debug Stats", &statsWindowOpen, ImGuiWindowFlags_NoFocusOnAppearing);
```

Comment:
might be nice to add something to the PR description explaining this change

Reply:
It's this part from the OP: `Also added the ability to close the windows with the "X" button to windows that were missing it.`

Reply:
coolcool, makes sense!